app/observers/manual_observers_registry.rb

```diff
@@ -48,6 +48,7 @@ def publication_logger
     ->(manual, _: nil) {
       manual.documents.each do |doc|
         next unless doc.needs_exporting?
+        next if doc.minor_update?
```

Comment:
I don't know if there's an existing test for this functionality, but I think it'd be good if there was one to make it clear this is expected behaviour.